docker-compose-services/elasticsearch/README.md

```diff
@@ -8,11 +8,11 @@ Copy [docker-compose.elasticsearch.yaml](docker-compose.elasticsearch.yaml) to y
 
 ## Configuration
 
-From within the container, the elasticsearch container is reached at hostname: elasticsearch, port: 9200, so the server URL might be `http://elasticsearch:9200`.
+From within the container, the elasticsearch container is reached at hostname: elasticsearch, port: 9200, so the server URL might be `http://elasticsearch:9200`
 
 ## Connection
 
-You can access the Elasticsearch server directly from the host for debugging purposes by visiting `http://<DDEV_SITENAME>.ddev.site:9200`
+You can access the Elasticsearch server directly from the host for debugging purposes by visiting `http://<DDEV_SITENAME>.ddev.site:9200`. If you have SSL enabled, which is recommended, you can access Elasticsearch via `https://<DDEV_SITENAME>.ddev.site:9201`
```

Comment:
You can also access it from both of those URLs inside the web container. So if people think the webserver should talk to elasticsearch via https they can use this URL. Probably a hair slower than using `http://elasticsearch: 9200`

Reply:
> You can also access it from both of those URLs inside the web container

I tried that, but it didn't work for me.

Reply:
![image](https://user-images.githubusercontent.com/125707/78719862-4ec14680-792d-11ea-85b7-76e848eff280.png)

Reply:
I didn't say you could use https://elasticsearch. I said you could use https://<projectname>.ddev.site:9201 - that goes through the router. The elasticsearch container you're using does not have mkcert respect or https built into it.